From 77503a0cefc4a1f5cccc56762d4a0fdc9c476957 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Tue, 17 Jul 2012 18:43:22 +0800
Subject: [PATCH 028/310] ENGR00217434 sabersd:auto build include gms if have
 the package.

add a if condition check whether have gms package on your build env,
if have, auto include it.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx6/BoardConfigCommon.mk | 2 +-
 imx6/sabresd_6dq.mk       | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/imx6/BoardConfigCommon.mk b/imx6/BoardConfigCommon.mk
index af754a0..4869fc5 100644
--- a/imx6/BoardConfigCommon.mk
+++ b/imx6/BoardConfigCommon.mk
@@ -51,5 +51,5 @@ PREBUILT_FSL_IMX_GPU := true
 
 BOARD_RECOVERY_PARTITION_SIZE = 10M
 
-BOARD_SYSTEMIMAGE_PARTITION_SIZE := 209715200
+BOARD_SYSTEMIMAGE_PARTITION_SIZE := 293601280
 BOARD_FLASH_BLOCK_SIZE := 4096
diff --git a/imx6/sabresd_6dq.mk b/imx6/sabresd_6dq.mk
index 6d0c567..c2a6ecb 100644
--- a/imx6/sabresd_6dq.mk
+++ b/imx6/sabresd_6dq.mk
@@ -2,6 +2,7 @@
 # It will inherit from FSL core product which in turn inherit from Google generic
 
 $(call inherit-product, device/fsl/imx6/imx6.mk)
+$(call inherit-product-if-exists,vendor/google/products/gms.mk)
 
 # Overrides
 PRODUCT_NAME := sabresd_6dq
-- 
1.8.0

